use-cases.html

```diff
@@ -0,0 +1,303 @@
+
+          <p>
+            A user finds themselves in an unfamiliar area. They want to check their
+            position so they use their handheld device to navigate to a web-based
+            mapping application that can pinpoint their exact location on the map
+            using the Geolocation Sensor API. They then ask the web application to provide
+            driving directions from their current position to their desired destination,
+            essentially responding to the question "Where am I going?".
+          </p>
+        </section>
+
+      </section>
+
+      <section id="usecases_continuous_background">
+        <h3>Background—Continuous geolocation updates</h3>
+
+        <section id="usecase_social_networking">
+          <h4>Location-tagged status updates in social networking applications</h4>
+
+          <p>
+            A social networking application allows its users to automatically tag their
+            status updates with location information. It does this by monitoring the
+            user's position with the Geolocation Sensor API. Each user can control the
+            granularity of the location information (e.g., city or neighborhood level)
+            that is shared with the other users. Any user can also see their network
+            of friends and get real-time updates about their current location,
+            granted they have opted in to their location data being shared.
+          </p>
+        </section>
+
+        <section id="usecase_turn_by_turn">
+          <h4>Turn-by-turn route navigation</h4>
+
+          <p>
+            A mapping application can help the user navigate along a route by
+            providing detailed turn-by-turn directions. The application does this by
+            registering with the Geolocation Sensor API to receive repeated location updates
+            of the user's position. These updates are delivered as soon as the
+            implementing user agent determines that the position of the user has
+            changed, which allows the application to anticipate any changes of
+            direction that the user might need to do.
+            The application can be in the foreground, but likewise can be backgrounded,
+            for example, when the user turns their device off to save battery
+            on a long highway route section without side roads.
+          </p>
+        </section>
+
+        <section id="usecase_sports_tracking">
+          <h4>Tracking sports activity</h4>
+
+          <p>
+            A web application can track a user's sports activity, for example,
+            a marathon run or a bicycle race. Therefore, the application does not
+            need to be on the screen, but would be backgrounded while the user
+            performs their activity, maybe with their handheld device strapped to their arm.
+          </p>
+        </section>
+
+        <section id="usecase_real_estate">
+          <h4>Real estate search</h4>
```

Comment:
This feels like more of a geofencing use case but that depends on how many simultaneous geofences the API allows an application to set.

Reply:
That is a good point. Related, we should capture somehow that a high number of geofences might create a bad UX. Needs research whether the spec should set some limit or leave that up to the implementation.

Reply:
If we imagine there is a high hit rate of matching real estate offers, then it should *not* be a geofencing use case, but a *"send location to a server in the background and have the server apply some business logic"* use case.

I agree that the spec should point out that implementations commonly enforce a limit of geofences that can be set, and that the ability to set a geofence is a privilege.

Reply:
I will clarify this aspect.